elections/2021/SC/README.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Knative Steering Committee Election - 2021 Voters Guide"
+linkTitle: "2021 Voters Guide"
+weight: 30
+type: "docs"
+---
+
+# 2021 Voters Guide - Knative Steering Committee Election
+
+## Purpose
+
+The role of this election is to fill out the three (3) seats due for election
+this year on the [Knative Steering Committee]. Each elected member will serve a
+two (2) year term.
+
+## Background
+
+This election will shape the future of Knative as a community and project. The
+Knative Steering Committee (SC) is responsible for the general health of the
+Knative community. Seats up for the election are the remaining bootstrap seats
+and in addition to contributor seats, we're introducing one (1) end-user seat to
```

Comment:
Do we want to make sure https://github.com/knative/community/issues/731 gets merged first?

Reply:
Also, based on #731 - the proposed change is that we have the 3 remaining bootstrap seats up for election. This change would mean we would have 2 of these seats be contributor seats and 1 of these seats be the end-user seat.

Reply:
This is mosdef wip still and gated on that, so yupp